.idea/codestyles/Project.xml

```diff
@@ -29,7 +29,7 @@
     <JetCodeStyleSettings>
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value>
-          <package name="kotlinx.android.synthetic" withSubpackages="true" static="false" />
+          <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />
```

Comment:
Is this an intential change this time? :D

Reply:
Yep!